spring-cloud-gcp-security-iap/src/main/java/org/springframework/cloud/gcp/security/iap/AppEngineAudienceValidator.java

```diff
@@ -32,13 +32,17 @@
  */
 public class AppEngineAudienceValidator extends AudienceValidator {
 
+	public static final String AUDIENCE_FORMAT = "/projects/%s/apps/%s";
```

Comment:
Does it make sense to make this as `public`?

Otherwise you need to add a Javadoc on it

Reply:
One more time: why do we need to have it as `public`? Doesn't look like some useful API for end-users of the framework.

Reply:
Ah, I took your original comment to mean that these constants should either be private or have useful javadoc. It's not a problem, I'll make them private.